src/theme/numberInput.ts

```diff
@@ -0,0 +1,28 @@
+const numberInput = {
+  components: {
+    NumberInput: {
+      baseStyle: {
+        field: {
+          borderRadius: "5px",
+          marginLeft: "2px",
+          marginRight: "2px",
+          maxW: "55px",
```

Comment:
specify it in the call site. it depends on the total number of digits each indidivual number picker supports.